web/src/actions/auth.js

```diff
@@ -135,7 +135,7 @@ const setupTokenManager = () => (dispatch, getState) => {
       // called when the token is closer to expiring
```

Comment:
@tbolt Why was this code change included in the money formatting issue? This doesn't appear to be related.

Reply:
It's formatting updates based on our `prettier js` rules